### ISSUE TITLE
Thread '<unnamed>' panicked at 'Value must reside in 1 or 2 registers'

### ISSUE THREAD

**Issue by luca-yvr** [Cardinal-Cryptography/aleph-node#594]
**Describe the bug**

node panic on restart with: Thread '<unnamed>' panicked at 'Value must reside in 1 or 2 registers'

**To Reproduce**

- this is aleph-node first crash since node was started almost 2 weeks ago
- I was adjusting vCPU number (on a different VM) around the time on the ESXi host, I'll be looking for relevant logs on the ESXi host as well - _edit: done that, nothing out of the ordinary in these logs_

**Environment information:**

 - OS:  Ubuntu 22.04.1 LTS
 - RAM:  16 GB
 - Cores:  10 vCores
 - Type:  VM on ESXi-7.0U3-18644231-standard
 - version: 34248bd (HEAD, tag: r-7.1, origin/release-7) Perform word splitting on custom arguments (#560)

**Additional context**

to summarize, there were 2 crashes:
1. at 16:38:43 - the node was synced and operating normally - **Thread 'tokio-runtime-worker' panicked at 'arithmetic operation overflow',**
2. at 18:58:29 - the node had just been restarted - **Thread '<unnamed>' panicked at 'Value must reside in 1 or 2 registers'**

after the 2nd crash, the VM was rebooted and node was restarted without issue

attaching node log (2nd crash)

[aleph-node-2.log](https://github.com/Cardinal-Cryptography/aleph-node/files/9375211/aleph-node-2.log)

**Comment by luca-yvr** [comment]
adding logs from 1st crash

[qnode-ca-2_node_crash_-_aug-17_at_16-38-43.zip](https://github.com/Cardinal-Cryptography/aleph-node/files/9375929/qnode-ca-2_node_crash_-_aug-17_at_16-38-43.zip)

**Comment by fixxxedpoint** [comment]
This one looks difficult to investigate. Could you provide us little bit more details: how did you obtained aleph-node's binary - using our docker image, or built manually. What is your cpu's architecture/model? If you built it manually, did you use for it same machine and system on which you run your node?

**Comment by luca-yvr** [comment]
built from source (git checkout r-7.1) on the same VM that aleph-node is currently running on.   ESXi host: MSI MS-7B79, AMD Ryzen 7 3800X 8-Core Processor, Memory 64GB, 2TB Samsung 970 EVO Plus NVMe, 10Gbps intel nic.  this node (QNODE-CA-2) is also on polkadot telemetry.  As a side note, I'm plannig to move this ESXi host to more substantial hardware (Epyc) within a couple of weeks

**Comment by fixxxedpoint** [comment]
I am closing it, since we are unable to reproduce it. We tried to run it on some AMD machine, hoping it might be somehow related (cpu features etc.). We also tried to investigate it based on provided logs but couldn't find what might be causing it.

**Comment by luca-yvr** [comment]
this never problem re-occurred at my end either.  I since moved to r7.2, and just yesterday migrated the node to different hardware.
